### PR TITLE
Document that ethers doesn't use Hardhat Network's config values for gas, gasPrice, gasMultiplier

### PR DESCRIPTION
**Pull request by feuGeneA**

### CODE REVIEW

File: docs/hardhat-network/reference/README.md [hunk 1]
```diff
@@ -28,14 +28,20 @@ The address to use as default sender. If not present the first account of the Ha
 
 Its value should be `"auto"` or a number. If a number is used, it will be the gas limit used by default in every transaction. If `"auto"` is used, the gas limit will be automatically estimated. Default value: the same value as `blockGasLimit`.
 
+Note that when using `ethers` this value will not be applied.
+
 #### `gasPrice`
 
 Its value should be `"auto"` or a number (in wei). This parameter behaves like `gas`. Default value: `"auto"`.
 
+Note when when using `ethers` this value will not be applied.
```
Comment (fvictorio) [inline docs/hardhat-network/reference/README.md:37]:
```suggestion
Note that when using `ethers` this value will not be applied.
```

**Review by fvictorio — APPROVED** [review]
Small comment, but LGTM.

**Comment by alcuadrado** [comment]
I think we should add a section explaining this to the ethers plugin's readme

File: packages/hardhat-ethers/README.md [hunk 2]
```diff
@@ -129,3 +129,7 @@ To create a contract factory, all libraries must be linked. An error will be thr
 Ethers.js polls the network to check if some event was emitted (except when a `WebSocketProvider` is used; see below). This polling is done every 4 seconds. If you have a script or test that is not emitting an event, it's likely that the execution is finishing before the event is detected by the polling mechanism.
 
 If you are connecting to a Hardhat node using a `WebSocketProvider`, events should be emitted immediately. But keep in mind that you'll have to create this provider manually, since Hardhat only supports configuring networks via http. That is, you can't add a `localhost` network with a URL like `ws://localhost:8545`.
+
+### Gas transaction parameters in `hardhat.config` are not used
+
+When using this plugin, the `gas`, `gasPrice` and `gasMultiplier` parameters from your `hardhat.config` are not automatically applied to transactions. In order to provide such values to your transactions, specify them as overrides on the transaction itself.
```
Comment (fvictorio) [inline packages/hardhat-ethers/README.md:135]:
```suggestion
When using this plugin, the `gas`, `gasPrice` and `gasMultiplier` parameters from your `hardhat.config` are not automatically applied to transactions. In order to provide such values to your transactions, specify them as [overrides](https://docs.ethers.io/v5/single-page/#/v5/api/contract/contract/-%23-contract-functionsSend) on the transaction itself.
```

**Pull request merged**